src/app/dim-ui/CircleProgress.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styles from './DiamondProgress.m.scss';
+
+interface Props {
+  /** 0-1 progress for the outer ring */
+  progress: number;
+  /** Level to display */
+  level?: number;
+  /** The icon to use */
+  icon: string;
+  icon2?: string;
+  className?: string;
+}
+
+/**
+ * A circle-shaped progress bar (from faction icons).
+ */
+export default function CircleProgress({ progress, level, icon, icon2, className }: Props) {
+  const style = {
+    strokeDashoffset: 121.622368 - 121.622368 * progress,
```

Comment:
How'd we arrive at these numbers?

Reply:
They are the same as the diamond progress numbers. We spot checked and it seemed to work for the circles as well. So however you came up with the diamond numbers.

Reply:
Nice